packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -31,12 +29,12 @@ import 'build.dart';
 /// managers.
 class BuildIOSFrameworkCommand extends BuildSubCommand {
   BuildIOSFrameworkCommand({
-    FlutterVersion flutterVersion, // Instantiating FlutterVersion kicks off networking, so delay until it's needed, but allow test injection.
-    @required BuildSystem buildSystem,
-    @required bool verboseHelp,
-    Cache cache,
-    Platform platform
-  }) : _flutterVersion = flutterVersion,
+    FlutterVersion? flutterVersion, // Instantiating FlutterVersion kicks off networking, so delay until it's needed, but allow test injection.
```

Comment:
Can optional named arguments be annotated with `@visibleForTesting`?

Reply:
Yes, good idea.